staging/src/k8s.io/apiserver/pkg/server/genericapiserver.go

```diff
@@ -137,9 +140,15 @@ type GenericAPIServer struct {
 	// listedPathProvider is a lister which provides the set of paths to show at /
 	listedPathProvider routes.ListedPathProvider
 
-	// DiscoveryGroupManager serves /apis
+	// DiscoveryRootManager serves /apis
+	DiscoveryRootManager discoveryroot.RootManager
+
+	// DiscoveryGroupManager serves /apis in an unaggregated form
 	DiscoveryGroupManager discovery.GroupManager
 
+	// DiscoveryResourceManager serves /apis in an aggregated form
+	DiscoveryResourceManager discoveryendpoint.ResourceManager
```

Comment:
Hrm, I might call this `AggregatedDiscoveryGroupManager` for clarity and `discoveryendpoint.RootManager`.  Resource kind of makes sense, but it's not really connected as well to the feature.